### PR TITLE
Friends TV Show CAH

### PR DESCRIPTION
**Pull request by Madhav-byte-debug**
Friends is very popular show, and I am sure many severs will use this pack.

### CODE REVIEW

File: deck_friends.go [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+			&PromptCard{Prompt: `According to Chandler, this phenomenon scares the bejesus out of him`},
+			&PromptCard{Prompt: `Hello? Vega, yeah we would like some alcohol, and you know what else? We would like some more %s.`},
+			&PromptCard{Prompt: `What makes me feel like a perfect arse?`},
+			&PromptCard{Prompt: `Stop it! You're killing me! I think I just moved on to phase four: %s!`},
+			&PromptCard{Prompt: `The question should be, what is *not* so great about %s. Ok? And the answer would be nothing!`},
+			&PromptCard{Prompt: `WHAT ARE YOU DOING TO MY SISTER?`},
+			&PromptCard{Prompt: `I'll be %s. When the rain starts to fall. `},
+			&PromptCard{Prompt: `I didn't have to, because I was wearing my I heart %s sandwich board and ringing my bell. `},
+			&PromptCard{Prompt: `I think Chandler would have achieved phase 3 faster if his friends had tried %s. `},
+			&PromptCard{Prompt: `MORRRNING'S HEEEERE, %S is here.`},
```
Comment (l-zeuch) [inline deck_friends.go:19]:
```suggestion
			&PromptCard{Prompt: `MORRRNING'S HEEEERE, %s is here.`},
```
A typo perhaps?

**Review by l-zeuch — CHANGES_REQUESTED** [review]

**Review by Madhav-byte-debug — COMMENTED** [review]
Yeah sorry for that. Reviewed the code again. No errors anymore I believe.

**Comment by l-zeuch** [comment]
> Yeah sorry for that. Reviewed the code again. No errors anymore I believe.

no problem :-), code lgtm!

A very nice pack...

**Review by l-zeuch — APPROVED** [review]
Just for the gesture 👍

**Pull request merged**